packages/base/core/src/less/_variables.less

```diff
@@ -1835,6 +1835,12 @@
 @oui-status-background-color_warning: @warning-bg-color;
 @oui-status-background-color_error: @error-bg-color;
 
+// Status - S
+@oui-status-s-line-height: 1.25;
+@oui-status-s-padding: 0 rem-calc(8) ;
+@oui-status-s-font-size: rem-calc(12);
+@oui-status-s-font-weight: @oui-font-bold;
```

Comment:
I know it's what it's asked by the mockup, but I found the use of `bold` a bit too much for the small variant. Especially when the bigger variant use `semibold`. What do you think ?

Reply:
I personally disagree, since `semibold` looks too thin on smaller `font-size`, so the way to solve the a11y ambiguity might be to set a `bold` `font-weight`